extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -103,36 +100,19 @@ export class Toolbar {
     this.toolbarClone_ = this.toolbarDOMElement_.cloneNode(true);
     const targetId = this.toolbarDOMElement_.getAttribute('target');
     // Set the target element to the toolbar clone if it exists.
-    const targetElement =
-    this.win_.document.getElementById(`${targetId}_toolbar`) ||
-    this.win_.document.getElementById(targetId);
-    this.targetElement_ = targetElement || this.createTargetElement_();
-    this.targetElement_.appendChild(this.toolbarClone_);
-    toggle(this.targetElement_, false);
-
-    // Check if the target element was created by us, or already inserted by the user
-    if (!this.targetElement_.parentElement) {
+    const targetElement = this.win_.document.getElementById(targetId);
+    if (targetElement) {
+      this.targetElement_ = targetElement;
       this.toolbarClone_.classList.add(TOOLBAR_ELEMENT_CLASS);
-      const fragment = this.win_
-        .document.createDocumentFragment();
-      fragment.appendChild(this.targetElement_);
-      this.body_.appendChild(fragment);
+      this.targetElement_.appendChild(this.toolbarClone_);
+      toggle(this.targetElement_, false);
+    } else {
+      throw new
```

Comment:
Since the error is deferred here, I recommend then to explicate this in comments. E.g. "This error will be later rethrown as a user error and the side bar will continue to function w/o toolbar feature".